sql/information_schema/columns_table.go

```diff
@@ -163,6 +168,7 @@ func (c ColumnsTable) WithDefaultsSchema(sch sql.Schema) (sql.Table, error) {
 		return nil, sql.ErrInvalidChildrenNumber.New(c, len(sch), len(c.allColsWithDefaultValue))
 	}
 
+	// TODO: generated values
```

Comment:
do you wanna do this one here? the info schema and show queries might expect resolved generate expressions

Reply:
I want to get what I have working in before coming back to do this, I'm 80% sure this would work but I haven't written the tests yet

Reply:
just tryna make sure info schema doesn't panic when people specify a virtual column